Linux/Ubuntu-20/Base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:20.04
+
+# To make it easier for build and release pipelines to run apt-get,
+# configure apt to not require confirmation (assume the -y argument by default)
+ENV DEBIAN_FRONTEND=noninteractive
+RUN echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
+
+RUN apt-get update \
+&& apt-get install -y --no-install-recommends \
+        ca-certificates \
+        curl \
+        jq \
+        git \
+        iputils-ping \
+        libcurl4 \
+        libunwind8 \
```

Comment:
Do we know that this doesn't need libicu as per docs & ubuntu 18?

Reply:
Added libicu66 as supported on Ubuntu 20